modules/gltf/editor/editor_scene_importer_blend.cpp

```diff
@@ -276,6 +276,13 @@ static bool _test_blender_path(const String &p_path, String *r_err = nullptr) {
 #else
 	path = path.plus_file("blender");
 #endif
+
```

Comment:
are you sure this is correct? you would get "path/blender/Blender" (since the plus_files above also apply).